draft-ietf-rtcweb-jsep.xml

```diff
@@ -2391,7 +2400,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             the most recent gathering phase (see
             <xref target="sec.ice-gather-overview"></xref>), an
             "a=candidate" line MUST be added, as defined in
-            <xref target="RFC5245"></xref>, Section 4.3., paragraph 3.
+            <xref target="8445"></xref>, Section 4.1.
```

Comment:
This reference doesn't seem right - there is no S 4.1 in 8445.

Reply:
Seems like this should be a ref to sip-sdp

Reply:
Yep; fixed